QualityControl/public/common/filters/runMode/runModeCheckbox.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+
+/**
+ * Render a run mode switch
+ * @param {object} filterModel the filter model
+ * @param {object} viewModel the model of the view
+ * @returns {Component} the run mode switch component
+ */
+export const runModeCheckbox = (filterModel, viewModel) => {
+  const { active } = viewModel.model.loader;
```

Comment:
Not sure how reliable the loader is. Do we need to disable the switch? It is not a heavily resource operation